schemas/documents/shared/gettingStartedResourcesSection.js

```diff
@@ -0,0 +1,26 @@
+export default {
```

Comment:
set this as a document so a same section with the same cards can be reused on multiple pages
this is currently the case on the homepage, all projects page and individual project pages